source/postcard-rpc/src/server/impls/embassy_usb_v0_3.rs

```diff
@@ -163,7 +163,7 @@ pub mod dispatch_impl {
             driver: D,
             config: Config<'static>,
             tx_buf: &'static mut [u8],
-        ) -> (UsbDevice<'static, D>, WireTxImpl<M, D>, WireRxImpl<D>) {
```

Comment:
Ah, did you mean to change this? This is also a breaking change (which I want to avoid if possible), and currently breaks CI.

If you want this interface, can you add it as a different (additional) constructor, instead of changing the existing one?

Reply:
Ah, I completely forgot about this. Thanks for pointing it out!